app/models/video.rb

```diff
@@ -1,11 +1,19 @@
 class Video < ActiveRecord::Base
 
+  has_many :reviews, ->{ order("created_at DESC") }
+  has_many :queue_items, ->{ order(:list_position)}
```

Comment:
Added spacing before }, I'm appreciate you catching these things. I think it's a good idea for code to be easy to read.